uint8 LWI_Sensorstatus
uint8 LWI_StrWhlAngleSt
float32 LWI_StrWhlAngleSize 
uint8 LWI_StrWhlAngleDrt
uint8 LWI_StrWhlSpeedDrt
float32 LWI_StrWhlSpeedSize

float32 Stalls
float32 TargetStalls

uint8 EngineSpeedState
float32 EngineSpeed


uint8 ACCSignalContinuity
float32 Speed 
uint8 ESP_SystemStatus
uint8 SpeedState

float32 LF_WhlSpd
float32 LR_WhlSpd
float32 RF_WhlSpd
float32 RR_WhlSpd

float32 aPedalPercent
uint8 aPedalPercentSt
float32 ThrottleGradientSize
uint8 ThrottleGradientPN
uint8 EngNeutralTorque


uint8 BrakeSwitch

float32 EPSRxHCA_Status
float32 EPS_StrWhlTorque 
uint8 EPS_StrWhlTorqueDrt
uint8 EPS_StrWhlTorqueSt

float32 ExpStrWhlTorque
uint8 HCATranCycle
uint8 ExpStrWhlTorqueDrt
float32 HCA_Status

float32 EngTorqueCoefficient

float32 ThtottlePosition
float32 aPedalPosition

float32 PLA_CRC
float32 PLA_BZ
uint8 PLABrkRqtSt
float32 PLAExpStrWhlAngle
uint8 PLAExpStrWhlAngleDrt
float32 PLARequestStatus
float32 PLABrkTorque
float32 PLABrkDeceleration
uint8 PLABrkEnable
uint8 BrkTrqAndDeceSwt
uint8 PLAParking
float32 PLAPrkDistance
uint8 PLASignalTxCyclic

uint8 YawState
uint8 axState
uint8 ayState
float32 ay 
float32 ax
float32 YawRate
uint8 YawToRight

uint8 BrakePressureState
float32 BrakePressure
float32 LF_WhlSpdDrt
float32 LR_WhlSpdDrt
float32 RF_WhlSpdDrt
float32 RR_WhlSpdDrt



float32 ACC_02CRC
float32 ACC_02BZ
float32 ExpectedSpeed  
float32 SpacingFactor
float32 NoiseAlarm
float32 SetSpacing
uint8 ImageAlarm
float32 ACC02Status


float32 ACC_06CRC
float32 ACC_06BZ
float32 aLowerDeviation
float32 aExpectedValue 
float32 aUpperDeviation
float32 DclrtGradRmdValue
float32 AclrtGradRmdValue
uint8 StartRequest06
uint8 ParkingRequest06
float32 ACC06Status

float32 ACC_07CRC
float32 ACC_07BZ
float32 ParkingDistance
uint8 ParkingRequest07
uint8 StartRequest07
uint8 aExpValue

float32 EPBFaultStatus
float32 EPBSwitch
uint8 EPBSwitchState
float32 EPB_PressingForce
float32 EPB_Status

float32 TSK_status


